targets/csharp/source/PlayFabSDK/source/Uunit/tests/PlayFabServerApiTest.cs

```diff
@@ -237,6 +240,60 @@ public void ParallelRequest(UUnitTestContext testContext)
                 }
             });
         }
+
+        /// <summary>
+        /// Multiplayer API
+        /// Try to deliberately request a mutliplayer alias that doesn't exist 
+        ///   Verify that response is MutliplayerServerNotFound, not just Not Found
+        /// </summary>
+        [UUnitTest]
+        public async void TestForNotFoundWithImportantInfo(UUnitTestContext testContext)
+        {
+            PlayFabSettings.staticSettings.TitleId = testTitleData.titleId;
+            PlayFabSettings.staticSettings.DeveloperSecretKey = testTitleData.developerSecretKey;
+
+            var eReq = new AuthenticationModels.GetEntityTokenRequest();
+            eReq.Entity = new AuthenticationModels.EntityKey();
+            eReq.Entity.Type = "title";
+            eReq.Entity.Id = testTitleData.titleId;
+
+            var tokenTask = await PlayFabAuthenticationAPI.GetEntityTokenAsync(eReq);
+
+            var staticPlayer = PlayFabSettings.staticPlayer;
+
+            if(tokenTask.Error != null)
+            {
+                testContext.Fail("Failed to retrieve the Title Entity Token, check your playFabSettings.staticPlayer, are they still logged in? (hint no server api should be called through a logged in client)");
```

Comment:
Instead of sending messages to your future self, just write the debug info into the message here.
Appending... staticPlayer.entityType would probably be enough.

Reply:
Also, there's better ways to do checks like this:
testContext.IsNull(tokenTask.Error, "message");